### ISSUE TITLE
ff_transactions.flea_conn times out

### ISSUE THREAD

**Issue by WendiGhost** [ffverse/ffscrapr#356]
**Describe the bug**
ff_transaction fails to pull transaction history from fleaflicker

**Reprex**
Please provide a short, reproducible example of the code you are trying to run. 

library(ffscrapr)
library(dplyr)
library(tidyr)

conn <- fleaflicker_connect(season = 2021, league_id = 326798)
tx <- ff_transactions(conn)

```r
conn <- fleaflicker_connect(season = 2021, league_id = 326798)
tx <- ff_transactions(conn)
```
**Expected behavior**
return league's transaction history in a df

**Session information**
R version 4.2.0 (2022-04-22 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.utf8  LC_CTYPE=English_United States.utf8    LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                           LC_TIME=English_United States.utf8    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.2.0 tools_4.2.0   
```r

```
Discord discussion link: https://discord.com/channels/550775812434165787/869777921231953951/981940350119719022

**Comment by WendiGhost** [comment]
This function still works with the example; for some reason not working for my league / settings. Wondering if its a volume / time out issue

**Comment by WendiGhost** [comment]
Anything I can do to prevent this from timing out?

**Comment by tanho63** [comment]
Resolved in #385.
